build-logic/publishing/src/main/kotlin/build-logic.publish-to-tmp-maven-repo.gradle.kts

```diff
@@ -0,0 +1,40 @@
+plugins {
+    id("maven-publish")
+}
+
+val localRepoElements by configurations.creating {
```

Comment:
do you have an actual use case for local publication? So far we did not have the need. We can still add this later on if we require it.

Reply:
It is useful to check if the publication works fine, including pom, jars, javadocs, metadata and so on.
I don't like to throw temporary files into mavenLocal, and I find it easier to find artifacts in `build/...` folder than in `.m2`.

Reply:
I see, good with me